mariannaFervenza/Quiz/Q1.py

```diff
@@ -0,0 +1,40 @@
+# Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of #size ‘k’.
+
+
+# So we are given an array of k = [positive numbers]
+# and we are given a variable k = "integer that is the size of the array"
+# We need to find the max sum of of this subarrays with size k
```

Comment:
I am missing a clear and concise high-level overview of how you will arrive at the solution. It is extremely important that the interviewer can have a glimpse into what you are thinking and how you organize your thoughts.